modules/apps/jaggery/mdm/src/mdm/units/login/login.hbs

```diff
@@ -8,7 +8,7 @@
         <span class="wr-validation-summary hidden">
                 <p>Login failed. Check your credentials and try again.</p>
         </span>
-        <form method="POST" class="form-login-box" action="{{loginPath}}">
+        <form id=login method="POST" class="form-login-box" action="{{loginPath}}" >
```

Comment:
unquoted attributes are valid on html5 specification, but as a best practice do recommend to use quotes.